library/java/org/chromium/net/impl/CronetBidirectionalStream.java

```diff
@@ -653,13 +653,13 @@ private void onErrorReceived(int errorCode, EnvoyFinalStreamIntel finalStreamInt
       mResponseInfo.setReceivedByteCount(finalStreamIntel.getReceivedByteCount());
     }
 
-    NetError netError = mapEnvoyMobileErrorToNetError(finalStreamIntel.getResponseFlags());
+    NetError netError = mapEnvoyMobileErrorToNetError(finalStreamIntel);
     int javaError = mapNetErrorToCronetApiErrorCode(netError);
 
     if (isQuicException(javaError)) {
       mException.set(new QuicExceptionImpl("Exception in BidirectionalStream: " + netError,
                                            javaError, netError.getErrorCode(),
-                                           /*nativeQuicError*/ 0));
+                                           /*nativeQuicError*/ 1));
```

Comment:
I did not do a mapping of the internal errorcodes as this is a separate task and is not essential at the moment too.
cc: https://docs.google.com/document/d/1SD5y3W9roKq15Nq9syTBiut1XV8zmzGvbpkFljeVAUc/edit#heading=h.z4neyin3z8oj